cli/ssh.go

```diff
@@ -85,6 +85,10 @@ func ssh() *cobra.Command {
 				Fetch: func(ctx context.Context) (codersdk.WorkspaceAgent, error) {
 					return client.WorkspaceAgent(ctx, workspaceAgent.ID)
 				},
+				FallbackTroubleshootingURL: client.URL.ResolveReference(&url.URL{
+					Path:     fmt.Sprintf("/templates/%s", workspace.TemplateName),
+					Fragment: "readme",
+				}).String(),
```

Comment:
This isn't really a troubleshooting page, why not just always link to our docs instead? If we don't have a documentation page on how to troubleshoot then we should wait until we have one before adding this feature.

Reply:
I simply mirrored the frontend behavior, but @bpmct created #5007 so I'll likely update this. 👍🏻